scripts/nomad-dev.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+require() {
+  if ! hash "$1" &>/dev/null; then
+    echo "'$1' not found in PATH"
+    exit 1
+  fi
+}
+
+require nomad
+require consul
+require bindle-server
+require traefik
+
+trap 'kill $(jobs -p)' EXIT
+
+echo "Starting consul..."
+consul agent -dev -bootstrap-expect 1 -client '0.0.0.0' &>/tmp/consul.log &
+
+echo "Starting nomad..."
+nomad agent -dev &>/tmp/nomad.log &
+
+echo "Starting traefik..."
+traefik --entryPoints.http.address=:8088 --providers.consulCatalog &>/tmp/traefik.log &
+
+echo "Starting bindle..."
+bindle-server --unauthenticated &>/tmp/bindle.log &
+
+echo "Waiting for nomad..."
+while ! nomad server members 2>/dev/null | grep -q alive; do
+  sleep 2
+done
+
+dotnet run --project src/Web \
+  --Scheduler:Driver=nomad \
+  --Bindle:Url=http://localhost:8080/v1 \
+  --Nomad:Driver=raw_exec \
```

Comment:
Shouldn't we be running nomad with root permissions and let Hippo decide the default value based on the OS it's being run from?

I feel like if we're on a Linux OS, we should be testing the exec driver because that's what we should be testing against.

https://github.com/deislabs/hippo/blob/fd5f7582127cdef49121c5a38b2674cd98507e15/src/Infrastructure/Jobs/NomadJob.cs#L28